src/mappers/booleanMapper.ts

```diff
@@ -0,0 +1,7 @@
+import { ValueMapper } from '../abstracts/ValueMapper';
+
+const removeTextAfterValidNumber = (value: string): string | undefined =>
```

Comment:
> @Siemienik what you think about create a `utils`/`helpers` folder ? problably `removeTextAfterValidNumber` will be used in #46 and #48

This util is redundant if you use `Number.parseFloat`

![image](https://user-images.githubusercontent.com/2893099/96430059-fafbb800-1201-11eb-9f6a-fda28e27e174.png)

Reply:
Aah okay :smiley:, i had no idea that parseFloat do that under the hood